10/umbraco-workflow/getting-started/approval-thresholds.md

```diff
@@ -0,0 +1,49 @@
+# Approval thresholds
+
+{% hint style="info" %}
+This feature requires a license - learn more about [Workflow's licensing model](https://umbraco.com/products/umbraco-workflow)
+{% endhint %}
+
+Workflow's default behaviour requires one member of an approval group to approve a pending task to advance it through the workflow. 
+
+The approval thresholds feature introduces configuration options to set the required number of approvals for a given workflow stage.
+
+The threshold options are:
+
+- **One**: Pending tasks require approval from any member of the assigned approval group. This is the default behaviour for all installations (trial and licensed).
+- **Most**: Pending tasks require approcal from an absolute majority of group members. For example, a group with three members requires two approvals and a group with four members requires three approvals.
+- **All**: Pending tasks require approval from all group members.
```

Comment:
How about we put these into a table instead?
I think that would make it a little easier to read.

Reply:
I'll add the tabs once the PR has been merged.